recipes/bazel/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-chmod u+x **/*.sh
+scl enable devtoolset-4 bash
```

Comment:
This is not allowed. If you want to have this, remove the dependency on gcc from your recipe.